docs/installation.md

```diff
@@ -0,0 +1,45 @@
+# Installation
+
+To install this application you should search for "Elastic Agent" in the GCP Marketplace and install it through the GCP Console.
+
+From the product listing you can choose the version of the application to install. We maintain `7.17.x` and `8.x` stack versions.
+
+By clicking "Configure" a guided process starts to collect some deploymenta details.
+
+You will need to choose:
+- the GKE cluster where to deploy the application: you can select an existing cluster or create a new one;
+- the namespace within the GKE cluster; selecting `default` will deploy the Application to the `default` namespace and the Elastic Agent in the (`kube-system`);
+- the application instance name (this is the name of the deployed application in your GKE cluster);
+- the Service Account of the application; **do not change this field**, a dedicated Service Account with required permissions will be created;
```

Comment:
We need to as per [documentation](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/schema.md#type-service_account):
> All service accounts need to be defined as parameters in schema.yaml.
> If you add a Kubernetes ServiceAccount as a resource in your manifest, the deployment fails with an authentication error, because the deployer doesn't run with enough privileges to create a Service Account by default.

We can improve the wording though, and add a specific section about the service account and it's permissions in the documentation.